src/app/components/presentational/products-details-view/products-details-view.component.ts

```diff
@@ -1,11 +1,17 @@
-import { Component, Input } from '@angular/core';
-import { Product } from '../../../modules/shared/types/product';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { OrderProduct } from 'src/app/modules/shared/types/product.order';
 
 @Component({
   selector: 'app-products-details-view',
   templateUrl: './products-details-view.component.html',
   styleUrls: ['./products-details-view.component.scss'],
 })
 export class ProductsDetailsViewComponent {
-  @Input() product!: Product;
+  @Input() product!: OrderProduct;
+  @Input() shoppingCartImageUrl: String =
+    'https://www.bing.com/images/search?view=detailV2&ccid=hd6cAKPA&id=A044F2E65A996F214578F90EC88C8F48BD14A299&thid=OIP.hd6cAKPAERPHDbVUFWRbyAHaHa&mediaurl=https%3A%2F%2Fimages.designtrends.com%2Fwp-content%2Fuploads%2F2015%2F11%2F27111316%2FShopping-Cart-Icons35.png&cdnurl=https%3A%2F%2Fth.bing.com%2Fth%2Fid%2FR.85de9c00a3c01113c70db55415645bc8%3Frik%3DmaIUvUiPjMgO%252bQ%26pid%3DImgRaw%26r%3D0&exph=1200&expw=1200&q=shopping+cart+png+icon&simid=608011243316977000&form=IRPRST&ck=19702DDA66BB9A3312DB37D868B833D5&selectedindex=1&ajaxhist=0&ajaxserp=0&vt=0&sim=11';
+  @Output() addToShoppingCart: EventEmitter<OrderProduct> = new EventEmitter();
+  @Output() deleteProduct: EventEmitter<string> = new EventEmitter();
+  @Output() navigateToProductList: EventEmitter<string> = new EventEmitter();
```

Comment:
issue: here is wrong, the emitted value is ```void```, not ```string```. Please check the others ones to be correct depending on what values you need to emit